src/test/java/nextstep/ladder/InputViewTest.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.ladder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.Assertions.in;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+class InputViewTest {
+
+    private static InputView inputView = InputView.getInputView();
+    String userNames = "pobi,honux,crong,jk";
```

Comment:
인스턴스 변수의 접근 제한자는 private으로 구현합니다.

```suggestion
    private String userNames = "pobi,honux,crong,jk";
```